nodejs/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+# fallback placeholder. This should come from the `nodejs-version-lookup` role to
+# determine the apps's preferred nodejs version.
+nodejs_version: 4.1.2
```

Comment:
Hmm.. good point. I think I need some refactoring here to avoid running `node/npm install` for other applications.